src/pages/Main/MainSlide/MainSlide.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Pagination, Navigation } from 'swiper';
+import MAIN_SLIDE_DATA from './MainSlideData';
+import styled from 'styled-components';
+import 'swiper/css';
+import 'swiper/css/pagination';
+import 'swiper/css/navigation';
+
+function MainSlide() {
+  return (
+    <Box
+      pagination={{
+        type: 'fraction',
+      }}
+      navigation={true}
+      modules={[Pagination, Navigation]}
+      className="mySwiper"
+    >
+      {MAIN_SLIDE_DATA.map(data => {
+        return (
+          <SwiperSlide key={data.id}>
+            <SlideImg key={data.id} src={data.image} />
+          </SwiperSlide>
+        );
+      })}
+
```

Comment:
```suggestion
      {MAIN_SLIDE_DATA.map(({id,image}) => {
        return (
          <SwiperSlide key={id}>
            <SlideImg key={id} src={image} />
          </SwiperSlide>
        );
      })}

```
이렇게 직관적으로 수정할 수 있습니다

Reply:
수정 완료 했습니다!